### PR TITLE
fix numerous grafana issues and enable console install

### PR DESCRIPTION
**Pull request by gene-redpanda**
* Adds support for using the console role added in RAC
* Fixes Grafana playbook so that it correctly reads from the metrics endpoint of nodeexporter
* Adds Grafana playbook to handle TLS

### CODE REVIEW

File: ansible/deploy-prometheus-grafana.yml [hunk 1]
```diff
@@ -39,7 +39,7 @@
         static_configs:
           - targets: "{{ groups['redpanda'] | map('extract', hostvars, 'inventory_hostname') | map('regex_replace', '^(.*)$','\\1:9644') | list  }}"
       - job_name: "node"
-        metrics_path: "/public_metrics"
+        metrics_path: "/metrics"
```
Comment (twmb) [inline ansible/deploy-prometheus-grafana.yml:42]: Why this change?
/public_metrics is meant to be for public consumption
/metrics is far too detailed

Comment (gene-redpanda) [reply ansible/deploy-prometheus-grafana.yml:42]:
```    prometheus_scrape_configs:
      - job_name: "redpanda"
        metrics_path: "/public_metrics"
        static_configs:
          - targets: "{{ groups['redpanda'] | map('extract', hostvars, 'inventory_hostname') | map('regex_replace', '^(.*)$','\\1:9644') | list  }}"
      - job_name: "node"
        metrics_path: "/public_metrics"
        metrics_path: "/metrics"```

Node is the scrape for nodeexporter, we're still scraping public_metrics for redpanda.

Comment (tmgstevens) [reply ansible/deploy-prometheus-grafana.yml:42]: Yes, see line 38

File: ansible/provision-basic-cluster.yml [hunk 2]
```diff
@@ -15,3 +15,15 @@
       ansible.builtin.include_role:
         name: redpanda.cluster.redpanda_broker
       when: not skip_node | default(false) | bool
+
+- name: Provision redpanda console
+  hosts: client
+  vars:
+    advertise_public_ips: true
+    redpanda_version: latest
+    install_console: true
```
Comment (r-vasquez) [inline ansible/provision-basic-cluster.yml:24]: Is there a PR to add the usage of this flag to our documentation repo?

File: ansible/provision-tiered-storage-cluster.yml [hunk 3]
```diff
@@ -33,3 +33,24 @@
...
+
+- name: Provision redpanda console
+  hosts: client
+  vars:
+    advertise_public_ips: true
+    redpanda_version: latest
+    install_console: true
+    enable_tls: true
+    create_demo_certs: true
+    handle_cert_install: true
```
Comment (r-vasquez) [inline ansible/provision-tiered-storage-cluster.yml:45]: [question]

Will this override what's passed via flags? I know these are not intended for production use. Just wondering how does the var override works

**Review by r-vasquez — APPROVED** [review]
LGTM, thanks!

File: ansible/provision-tiered-storage-cluster.yml (hunk 3 above)
Comment (gene-redpanda) [reply ansible/provision-tiered-storage-cluster.yml:45]: Command line -> hosts file -> playbook iirc. But I can guarantee command line is supreme.

File: ansible/provision-basic-cluster.yml (hunk 2 above)
Comment (gene-redpanda) [reply ansible/provision-basic-cluster.yml:24]: Good call

Comment (r-vasquez) [reply ansible/provision-basic-cluster.yml:24]: For other reviewers, here is the PR: https://github.com/redpanda-data/documentation-private/pull/1942

Thanks Gene :+1:
